lib/smart_proxy_remote_execution_ssh.rb

```diff
@@ -83,6 +84,11 @@ def validate_ssh_log_level!
         Plugin.settings.ssh_log_level = Plugin.settings.ssh_log_level.to_sym
       end
 
+      def validate_socket_path!
```

Comment:
This is only relevant if one of the ssh modes is used. Could we enforce this conditionally?

Reply:
Ping